daprdocs/content/en/reference/environment/_index.md

```diff
@@ -24,4 +24,5 @@ The following table lists the environment variables used by the Dapr runtime, CL
 | DAPR_HELM_REPO_PASSWORD | A password for a private Helm chart  |The password required to access the private Dapr helm chart. If it can be accessed publicly, this env variable does not need to be set| 
 | OTEL_EXPORTER_OTLP_ENDPOINT | OpenTelemetry Tracing | Sets the Open Telemetry (OTEL) server address, turns on tracing. (Example: `http://localhost:4318`) |
 | OTEL_EXPORTER_OTLP_INSECURE | OpenTelemetry Tracing | Sets the connection to the endpoint as unencrypted. (`true`, `false`) |
-| OTEL_EXPORTER_OTLP_PROTOCOL | OpenTelemetry Tracing | The OTLP protocol to use Transport protocol. (`grpc`, `http/protobuf`, `http/json`) |
\ No newline at end of file
+| OTEL_EXPORTER_OTLP_PROTOCOL | OpenTelemetry Tracing | The OTLP protocol to use Transport protocol. (`grpc`, `http/protobuf`, `http/json`) |
+| DAPR_COMPONENTS_SOCKET_FOLDER | Pluggable component | The location for the `.sock` file for the shared directory location socket |
```

Comment:
`DAPR_COMPONENTS_SOCKET_FOLDER` is apparently an older environment variable that's been replaced by   `DAPR_COMPONENTS_SOCKETS_FOLDER`.

This new environment variable is used by the Dapr runtime as well as the .NET, Go, & Java pluggable components SDKs.

The .NET and Java SDKs also support `DAPR_COMPONENTS_SOCKETS_EXTENSION` environment variable, which indicates the default file extension applied to socket files created by the SDKs.  This value defaults to `.sock`  for .NET and `sock` for Java.

Reply:
> The .NET and Java SDKs also support `DAPR_COMPONENTS_SOCKETS_EXTENSION` environment variable, 

I don't know if would make sense to add those to the main Dapr docs. I guess mentioning them with the caveat it is a per-SDK configuration and not a Dapr enforced behaviour.

Aside this, I don't really recall any other env. var. aside `DAPR_COMPONENTS_SOCKETS_FOLDER` (notice the socketS vs socket diff).

Reply:
We could be a bit more verbose on this explanation, as in:

```suggestion
| DAPR_COMPONENTS_SOCKET_FOLDER | Pluggable component | The location or path where Dapr will look for Pluggable Components Unix Domain Socket files. If unset this location defaults to `/tmp/dapr-components-sockets` |
```

Notice that we don't mention the `.sock` extension: the code in dapr/dapr doesn't really care about it, it tests all files that that location for their kind (socket or regular), regardless of their extension.  Even for SDKs, adding those extensions to the UDS they create is more decorative than functional. 

The proposed text neither puts a limit to the number of such files that might exist (vs singular _"the `sock` file"_)